python-threatexchange/threatexchange/cli/config_cmd.py

```diff
@@ -620,10 +633,39 @@ def execute_config(self, settings: CLISettings) -> None:
             settings.set_persistent_config(config)
 
 
+class ConfigNCMECAPICommand(command_base.Command):
+    """Configure NCMEC hash api integration"""
+
+    @classmethod
+    def get_name(cls) -> str:
+        return NCMECSignalExchangeAPI.get_name()
+
+    @classmethod
+    def init_argparse(cls, settings: CLISettings, ap: argparse.ArgumentParser) -> None:
+        ap.add_argument(
+            "--credentials",
+            metavar="STR",
+            nargs=2,
+            help="set the username and password to access the NCMEC API",
+        )
+
+    def __init__(
+        self,
+        credentials: t.List[str],
+    ) -> None:
+        self.credentials = (credentials[0], credentials[1]) if credentials else None
+
+    def execute(self, settings: CLISettings) -> None:
+        if self.credentials is not None:
+            config = settings.get_persistent_config()
+            config.ncmec_credentials = self.credentials
```

Comment:
Unfortunately, it's hard to directly validate on the command without changing the argparse format (nargs=2 treats each arg separately by default), but at least we could validate here